src/protocol/decoder.rs

```diff
@@ -270,11 +276,9 @@ impl TychoStreamDecoder {
 #[cfg(test)]
 mod tests {
     use crate::{
-        models::ERC20Token,
         evm::protocol::uniswap_v2::state::UniswapV2State,
-        protocol::{
-            decoder::{StreamDecodeError, TychoStreamDecoder},
-        },
+        models::ERC20Token,
+        protocol::decoder::{StreamDecodeError, TychoStreamDecoder},
     };
     use ethers::types::U256;
     use rstest::*;
```

Comment:
Why not use `use super::*;` here and skip re-importing many of these imports already used in this file?